spec/Section 2 -- Language.md

```diff
@@ -466,8 +466,9 @@ These two operations are semantically identical:
 
 Alias : Name :
 
-By default a field's response key in the response object will use that field's
-name. However, you can define a different response key by specifying an alias.
+:: A _response name_ is the key in the response object which correlates with a
```

Comment:
Thinking out loud -
Should `response object` be updated to `response map` in https://github.com/graphql/graphql-spec/pull/1039?

But then this will not be accurate with incremental delivery, as a `response name` can also appear in the subsequent payloads. Should there be a new term (Response Data Tree?) to refer to the entire response data?

Reply:
I think this is referring to the object in the response representing the current Object Type's data, as opposed to the entire response which may use multiple names/aliases which may or may not overlap. It's a good point though, we should be clearer about this!